services/cal/test/test_service.py

```diff
@@ -49,7 +49,6 @@ def setUpClass(self):
             self.cal_service = CalService.instance(CalMockRemote.instance())
             print('Mocking Remote...')
 
-    @classmethod
     def setUp(self):
```

Comment:
Why did you delete this ?

Reply:
I had read that it's only necessary for `setUpClass` and`tearDownClass` 
https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp